mars/services/storage/spill.py

```diff
@@ -60,7 +60,7 @@ def record_put_info(self, key, data_size: int):
         self._data_sizes[key] = data_size
 
     def record_delete_info(self, key):
-        self._data_sizes.pop(key)
+        self._data_sizes.pop(key, None)
```

Comment:
If `_data_sizes` can be missing, we also need to use `difference_update` in `self._spilling_keys.remove` below.

Reply:
It's already removed in next lines:
```python
if key in self._spilling_keys:
            self._spilling_keys.remove(key)
```